src/mediapipe_calculators/pytensor_ovtensor_converter.cc

```diff
@@ -0,0 +1,113 @@
+#include "src/mediapipe_calculators/pytensor_ovtensor_converter.pb.h"
+
+namespace py = pybind11;
+using namespace py::literals;
+using namespace ovms;
+
+namespace mediapipe {
+
+class PytensorOvtensorConverterCalculator : public CalculatorBase {
+    mediapipe::Timestamp outputTimestamp;
+
+public:
+    static absl::Status GetContract(CalculatorContract* cc) {
+        LOG(INFO) << "PytensorOvtensorConverterCalculator [Node: " << cc->GetNodeName() << "] GetContract start";
+        RET_CHECK(cc->Inputs().GetTags().size() == 1);
+        RET_CHECK(cc->Outputs().GetTags().size() == 1);
+        RET_CHECK((*(cc->Inputs().GetTags().begin()) == "OVTENSOR" && *(cc->Outputs().GetTags().begin()) == "OVMS_PY_TENSOR") || (*(cc->Inputs().GetTags().begin()) == "OVMS_PY_TENSOR" && *(cc->Outputs().GetTags().begin()) == "OVTENSOR"));
+        if (*(cc->Inputs().GetTags().begin()) == "OVTENSOR") {
+            cc->Inputs().Tag("OVTENSOR").Set<ov::Tensor>();
+            cc->Outputs().Tag("OVMS_PY_TENSOR").Set<PyObjectWrapper<py::object>>();
+        } else {
+            cc->Inputs().Tag("OVMS_PY_TENSOR").Set<PyObjectWrapper<py::object>>();
+            cc->Outputs().Tag("OVTENSOR").Set<ov::Tensor>();
+        }
+
+        LOG(INFO) << "PytensorOvtensorConverterCalculator [Node: " << cc->GetNodeName() << "] GetContract end";
+        return absl::OkStatus();
+    }
+
+    absl::Status Close(CalculatorContext* cc) final {
+        LOG(INFO) << "PytensorOvtensorConverterCalculator [Node: " << cc->NodeName() << "] Close";
+        return absl::OkStatus();
+    }
+
+    absl::Status Open(CalculatorContext* cc) final {
+        LOG(INFO) << "PytensorOvtensorConverterCalculator [Node: " << cc->NodeName() << "] Open start";
+        outputTimestamp = mediapipe::Timestamp(mediapipe::Timestamp::Unset());
+        LOG(INFO) << "PytensorOvtensorConverterCalculator [Node: " << cc->NodeName() << "] Open end";
+        return absl::OkStatus();
+    }
+
+    absl::Status Process(CalculatorContext* cc) final {
+        LOG(INFO) << "PytensorOvtensorConverterCalculator [Node: " << cc->NodeName() << "] Process start";
+        py::gil_scoped_acquire acquire;
+        if (*(cc->Inputs().GetTags().begin()) == "OVTENSOR") {
+            auto& inputTensor = cc->Inputs().Tag("OVTENSOR").Get<ov::Tensor>();
+            py::object outTensor;
+            std::unique_ptr<PyObjectWrapper<py::object>> outputPyTensor = std::make_unique<PyObjectWrapper<py::object>>(outTensor);
+            PythonBackend pythonBackend;
+            std::vector<py::ssize_t> shape;
+            for (const auto& dim : inputTensor.get_shape()) {
+                shape.push_back(dim);
+            }
+            const auto& options = cc->Options<PytensorOvtensorConverterCalculatorOptions>();
+            const auto tagOutputNameMap = options.tag_to_output_tensor_names();
+            auto outputName = tagOutputNameMap.at("OVMS_PY_TENSOR").c_str();
+            pythonBackend.createOvmsPyTensor(
```

Comment:
Unless we create copy we have the same issue, as in other conversion. We need to make sure that during  lifetime of created tensor underlying buffer is still allocated.